packages/pdf-generator/src/components/InsightsLabel.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View, Canvas, Text } from '@react-pdf/renderer';
+import styles from '../utils/styles';
+
+const appliedStyles = styles();
+
+//eslint-disable-next-line
```

Comment:
Leeeeeetsssss just do this once at  at the top of the file

/* eslint max-len: 0 */

Reply:
💯 🥇